src/lib/FormData.mjs

```diff
@@ -315,14 +315,14 @@ class FormData {
 
     // Set a new field if given name is not exists
     if (!field) {
-      return void this.__content.set(name, {
+      return void this.__content.set(String(name), {
```

Comment:
I do believe you can move all of these String(name) calls in just one call at the start of FormData#__setField method. Everything else looks fine. 👍

Reply:
I actioned that, and also fixed another place that was previously missed here:

https://github.com/octet-stream/form-data/pull/23/commits/fbfa47e347eb059421c4c65d5b6fa137b89e83e8#diff-97ef3ebef0da1767a9b59b755448daf1L314